packages/components/OmniLink.tsx

```diff
@@ -0,0 +1,61 @@
+import { useLinkProps } from "@react-navigation/native";
+import React, { ReactNode } from "react";
+import {
+  Platform,
+  StyleProp,
+  TouchableOpacity,
+  TouchableOpacityProps,
+  View,
+  ViewStyle,
+} from "react-native";
+
+export interface OmniLinkToType {
+  screen: string;
+  params?: object;
+}
+
+export const OmniLink: React.FC<{
+  to: OmniLinkToType;
+  action?: any | undefined;
+  children: ReactNode | undefined;
+  style?: StyleProp<ViewStyle | TouchableOpacityProps>;
+
+  disabled?: boolean;
+}> = ({ to, action, children, style, disabled, ...rest }) => {
+  // @ts-expect-error
```

Comment:
We should not hide error here

Reply:
This is a linter issue it will light up as a problem otherwise. 
It's a false positive.
And ideas how to get around it besides the existing lines?

Reply:
You can use `CustomPressable`.
Add `onMouseEnter?: () => void;` and `onMouseLeave?: () => void;` in `CustomPressableProps`.
Then you can replace this `View onClick` by `CustomPressable onPress`.

Also, we have this component : `packages/components/Link`.

Do you think you can deal with existing components ?

Reply:
After discussion, let's try like this !